rust/query-stats/README.md

```diff
@@ -0,0 +1,75 @@
+# Query stats
+
+## Prerequisites
+This example requires an installation of:
+
+- [x] Install the [IC SDK](../developer-docs/setup/install/index.mdx) at version 0.16.1 or higher.
+- [x] Download the following project files from GitHub: https://github.com/dfinity/examples/
+
+Begin by opening a terminal window.
+
+This is currently only for use from within DFINITY, as the replica shipped with `dfx` as well as nodes on 
+mainnet do not enable the query stats feature at the moment.
+
+Once enabled on mainnet, the following instructions will work without the `--network` argument.
+
+From within DFINITY, it can be ran as following (e.g. with a deployment to the `small_with_query_stats` Farm testnet)
+
+Variable `$BN_HOSTNAME` points at the hostname of a boundary node. For example, from field `['bn_aaaa_records']['url']` of the output of a Farm deployment
+
+## Installing the canister
+
+```
+dfx canister create --network="$BN_HOSTNAME" query_stats
+```
+
+We will need the canister ID that has been assigned to our canister later.
+
+```
+dfx build --network=$BN_HOSTNAME query_stats
+dfx canister install --network=$BN_HOSTNAME query_stats
+```
+
+## Issuing load
+
+We now want to generate queries at a certain rate. One way to achieve this is by using `ic-workload-generator`, which is part of the [IC repo](https://github.com/dfinity/ic/tree/master/rs/workload_generator). 
```

Comment:
Are there binaries available for downloading somewhere?

Reply:
No.
The workload generator is not strictly needed for this, but it's probably the easiest tool to get started.

What would you recommend instead? Should I recommend to run `dfx` calls in a bash loop instead?